googletest/src/gtest.cc

```diff
@@ -107,6 +107,10 @@
 # include <windows.h>  // NOLINT
 # undef min
 
+#elif GTEST_HAS_CHRONO_
```

Comment:
<chrono> is part of the C++ 11 standard so instead of GTEST_HAS_CHRONO probably want to use GTEST_LANG_CXX11 which is already a defined macro variable